src/test/java/org/openrewrite/staticanalysis/EqualsAvoidsNullVisitorTest.java

```diff
@@ -37,20 +37,26 @@ void invertConditional() {
           //language=java
           java(
             """
-              public class A {
-                  {
-                      String s = null;
-                      if(s.equals("test")) {}
-                      if(s.equalsIgnoreCase("test")) {}
-                  }
-              }
+                    public class A {
+                        {
+                            String s = null;
+                            if(s.equals("test")) {}
+                            if(s.equalsIgnoreCase("test")) {}
+                            System.out.println(s.compareTo("test"));
+                            System.out.println(s.compareToIgnoreCase("test"));
+                            System.out.println(s.contentEquals("test"));
+                        }
+                    }
               """,
```

Comment:
Let's not change the indentation here.
```suggestion
              public class A {
                  {
                      String s = null;
                      if(s.equals("test")) {}
                      if(s.equalsIgnoreCase("test")) {}
                      System.out.println(s.compareTo("test"));
                      System.out.println(s.compareToIgnoreCase("test"));
                      System.out.println(s.contentEquals("test"));
                  }
              }
              """,
```

Reply:
We have one in src/test/java which I would have thought would apply here:
https://github.com/openrewrite/rewrite-static-analysis/blob/692b242b9756b0b2290f56f8a05bfee0d95d5303/src/test/java/.editorconfig